docs/src/content/docs/basics/terminology.mdx

```diff
@@ -46,7 +42,7 @@ A container of data that holds the state for one aspect of a *connection*. The *
 
 An identity agent is an agent that represents the user on the internet. It manages the storage of personal information about the user. 
 
-Depending on the implementation variant (see *virtual identity agent* and *local identity agent*) this information may be stored on-device, and/or within an *app*, and/or by a third-party storage provider service. 
+Depending on the implementation variant (see *virtual identity agent* and *local identity agent*) this information may be stored within an *app*, and/or by a third-party storage provider service.
```

Comment:
Looks like the "App" term includes the information of being used on device, suggest removing this duplication

Reply:
I agree with this change.